### ISSUE TITLE
Not working for me in new Firefox build

### ISSUE THREAD

**Issue by Rocman76** [ArakawaHenri/iCloudPasswords_for_Firefox#5]
No longer works in the new Firefox build for me, I tried reinstalling and also  giving permission again, can you use the current Edge build to update?

**Comment by samophlange** [comment]
It seems updates can break it, it no longer works for me either.

**Comment by InvitedToHell** [comment]
Same for me

**Comment by TW527E** [comment]
Same for me

**Comment by ArakawaHenri** [comment]
I am really sorry but after arguing about it, Mozilla still refuses to sign the new version of the extension, which means I can't generate an xpi package for people to download.

**Comment by TW527E** [comment]
Is there a way for users to solve it by themselves?

**Comment by ChrisMBarr** [comment]
So it partially works for me actually.  It will autofill usernames but not passwords

**Comment by codecrete-ross** [comment]
So here's where we're at:

1. Mozilla half-bakes feature support
2. Mozilla allows developers spend their valuable time fully baking that support
3. Mozilla actively develops updates to break it
4. Mozilla refuses to let those volunteer developers fix what Mozilla broke

Mozilla has officially become a sad, petty, child of a company. They're upset that they're losing the browser war (that they should be winning) and refusing to do what's necessary to improve their footing.

Chromium browsers are becoming nearly unusable for power-users. Forced Efficiency Mode is crippling my daily workflows, so what's the natural course of action? Use an alternative browser, like Firefox. But wait, now I lose one of the most convenient and reliable password security features available to consumers, and the one option to regain it is getting a big "f*ck you" from Mozilla itself.

What an absolute _riot._

**Comment by InvitedToHell** [comment]
> So here's where we're at:
> 
> 
> 
> 1. Mozilla half-bakes feature support
> 
> 2. Mozilla allows developers spend their valuable time fully baking that support
> 
> 3. Mozilla actively develops updates to break it
> 
> 4. Mozilla refuses to let those volunteer developers fix what Mozilla broke
> 
> 
> 
> Mozilla has officially become a sad, petty, child of a company. They're upset that they're losing the browser war (that they should be winning) and refusing to do what's necessary to improve their footing.
> 
> 
> 
> Chromium browsers are becoming nearly unusable for power-users. Forced Efficiency Mode is crippling my daily workflows, so what's the natural course of action? Use an alternative browser, like Firefox. But wait, now I lose one of the most convenient and reliable password security features available to consumers, and the one option to regain it is getting a big "f*ck you" from Mozilla itself.
> 
> 
> 
> What an absolute _riot._

This is a philosophical masterpiece. It's a shame what happend to the world of browsers..

**Comment by atmanli** [comment]
Do we need to have a signed version?
Couldn’t you provide an unsigned version that we can install?

**Comment by ArakawaHenri** [comment]
Yes, you need a signed xpi file to install an addon permanently, otherwise you need to sideload unsigned addons each time you relaunch the browser.
btw, code for ver 1.3.93 pushed, if you want use it anyway, just sideload them everytime you restart FF and handle the configuration manually to grant the permission for interacting with the iCloud Client.

**Comment by codecrete-ross** [comment]
Can’t believe we’re still in this situation. I’m STILL wrestling with Edge. I’m going above and beyond trying to force this application to perform as it should, with limited success. 

Firefox would be gaining users if they spent one f*cking day solving this ridiculous problem.

**Comment by ArakawaHenri** [comment]
Actually, my suggestion would be just escape from FF. As it lacks so much on new feature support (HDR for Win, HEVC decoding even with hardware that can handle it natively) and such a limited addon ecosystem.
As a former free software promoter, Mozilla actually refuses to provide sufficient freedom to its users now, that is pathetic and irony.
If you are a normal user, just select any of those blink based browsers and it really brings you better experiences. And for opensource software lovers, Chromium could be a better choise.
 
Frankly speaking, FF is NEITHER a GOOD nor FREE browser today.

**Comment by ChrisMBarr** [comment]
Here's what I did recently, and it's my recommendation.  Move all your passwords out of iCloud and into [BitWarden](https://bitwarden.com/). BitWarden has apps for iOS, Mac, Windows, and Android. It has browser extensions for Chrome, Firefox, etc. It syncs between them all too, and since iOS allows for various password managers to be set up system wide, this seems to work pretty seamlessly. And it's free, but they do have paid plans for extra features, none of which I've found that I need.

**Comment by MoaiGuy** [comment]
Now if only apple would port the plug-in themselfs when they already got the bookmarks one covered, but then again i also wish that they would just have ported Safari again to windows, would have made it much easier.

**Comment by v-127** [comment]
> Here's what I did recently, and it's my recommendation. Move all your passwords out of iCloud and into [BitWarden](https://bitwarden.com/). BitWarden has apps for iOS, Mac, Windows, and Android. It has browser extensions for Chrome, Firefox, etc. It syncs between them all too, and since iOS allows for various password managers to be set up system wide, this seems to work pretty seamlessly. And it's free, but they do have paid plans for extra features, none of which I've found that I need.

Well, if I want to use iCloud passwords is because of how easy is to use it with iPhone, Mac and Windows with Chromium browsers... I don't understand why they would just not launch an extension for firefox browsers..
